### ISSUE TITLE
The comments duplicate will remain above the class after re-organizing.

### ISSUE THREAD

**Issue by naukri7707** [kokoabim/vscode-csharpen#1]
The comments duplicate will remain above class if I use `using` to import namespaces.

Here is my testing code:
```cs
using System;

public class Test : object
{
    // Some comments
    public void Foo()
    {
    }
}
```

After re-organizing, it will look like this:

```cs
using System;

// Some comments
public class Test : object
{
    // Some comments
    public void Foo()
    {
    }
}
```

But if there is no using, it will work perfectly.

```cs
public class Test
{
    // Some comments
    public void Foo()
    {
    }
}
```

After re-organizing: (nothing change in this case)

```cs
public class Test
{
    // Some comments
    public void Foo()
    {
    }
}
```

I also noticed that comments above the using directive get removed after reorganizing.
Maybe this is related to the issue?

```cs
// Some comments
using System;

public class Test : object
{
}
```

And after re-organizing, it becomes:

```cs
using System;

public class Test : object
{
}
```

This is a great project, appreciate your efforts.

**Comment by kokoabim** [comment]
Thanks. I'll take a look at it. I've found a couple other quirks lately that I need to address as well.

**Comment by kokoabim** [comment]
The code checked in tagged as 0.0.2 and soon its package to VS Marketplace _should_ have this fixed.

**Comment by naukri7707** [comment]
Hey, after further testing, I've found that this issue still persists.

I've retested the testing code I initially provided:

```cs
using System;

public class Test : object
{
    // Some comments
    public void Foo()
    {
    }
}
```

I discovered that this example was problematic. Since `System` wasn't being used, it would be removed by CSharpen, making it appear to work correctly. (I suspect this is because the removal of the `using` statement caused it to enter the state of the second example mentioned in the original post.)

Here's an updated testing code:

```cs
using System.Collections.Generic;

public class Test : List<int>
{
    // Some comments
    public void Foo()
    {
    }
}
```

This new testing code should consistently reproduce the error. I apologize for providing a problematic testing code initially.

**Comment by kokoabim** [comment]
In RE: to your initial example, I did have issues with the example code as you mentioned but I changed it a but and was able to reproduce (and resolve). But with your latest code, I can reproduce it as well. Could have been two similar issues. Will look into this latest example to resolve.

**Comment by kokoabim** [comment]
Found the issue of your last code example. It was related to do with `using` directives and the whitespace afterwards. This should be resolved with 0.0.3 that was just released.
